beacon-chain/sync/validate_blob.go

```diff
@@ -92,13 +93,46 @@ func (s *Service) validateBlob(ctx context.Context, pid peer.ID, msg *pubsub.Mes
 	switch parentStatus := s.handleBlobParentStatus(ctx, parentRoot); parentStatus {
 	case pubsub.ValidationIgnore:
 		log.WithFields(blobFields(blob)).Debug("Ignored blob: parent block not found")
+		if err := s.sendBatchRootRequest(ctx, [][32]byte{parentRoot}, rand.NewGenerator()); err != nil {
```

Comment:
sanity check - do we want to do a blocking RPC call here?

Reply:
doesnt make sense to block, ill move it to background routine